ui/react-components/components/Dropdown/AsyncDropdown.test.jsx

```diff
@@ -0,0 +1,214 @@
+import {fireEvent, waitForElement} from "@testing-library/react";
```

Comment:
Why do we have named some test files with `.spec` & `.test`? Are we using 2 different test frameworks?

Reply:
No, we are not. Both are allowed